### ISSUE TITLE
Localized Creature Say

### ISSUE THREAD

**Issue by Tulba** [AscEmu/AscEmu#110]
I have noticed the changes on creature say.

Are you working on localized creature say? I see the tables npc_monstersay_localized, npc_script_text_localized and npc_text_localized.

**Comment by Evairfairy** [comment]
Can you please be more specific about this? As far as I'm aware, localised text is supported (I can't verify this right now but I've noticed several areas of the core using localised texts)

**Comment by Zyres** [comment]
I changed a lot of the npc say "events".
For example:
Emote("You will die", Text_Yell, 1335); -> _unit->SendScriptTextChatMessage(444);        // You will die

The goal of these changes was to load all of these sounds, emotes, text from the databse. (Table npc_script_text, thx to tc). The function "sending localized text" is not ready yet.
To send multilingual text, we need the sessions of the players in an area/zone to send the correct text to him/her.
I found an example here: codesnippet (LocalizedWorldBroadCast nt;).
